src/replica/storage/simple_kv/test/case-207.act

```diff
@@ -64,77 +64,66 @@ client:replica_config:receiver=r1,type=downgrade_to_inactive,node=r1
 config:{4,-,[]}
 state:{}
 
-# assign primary r3
-client:replica_config:receiver=r3,type=assign_primary,node=r3
-config:{5,r3,[]}
-state:{{r3,pri,5,0}}
+# add primary r1
```

Comment:
I noticed that you update 3 unit tests related with learn process, and slao remove original comments. I am not sure will following questions:
- what the original case test for?
- what the difference between the original and tthe new one? can new case still work for the original case?

Could you please fulfill those answers in your pr descption and add comments about the modified process?

Reply:
Ok, let me explain why these three uts need to be changed,
If all replica are removed, and a new replica is created, ballot>0 will be found, which means that it has already existed before, so if a replica is created directly and the ballot starts from 0, it will lead to the loss of the previously existing data. I don't think this should have happened. All three uts are involved in this issue:
old-case-207:
```
=> state {{r1,pri,2,10},{r2,sec,2,10}}
remove sec r2 and remove pri r1
=> state {}
add new pri r3 and active sec r2
=> state{{r2,sec,6,0},{r3,pri,6,0}}
check r3 data
=> not-exist
change pri and check r2 data
=> not-exist
This case knows that the data is lost
So I restarted the downed replica to check if the data was lost:
=> state {{r1,pri,2,10},{r2,sec,2,10}}
remove sec r2 and remove pri r1
=>state{}
active pri r1 and check data
=> correct
add sec r2
{{r2,sec,6,10},{r1,pri,6,10}}
 chenge pri and check data
=> correct
```

old-case-208:
```
=> state:{{r1,pri,2,10,10},{r2,sec,2,10,10}}
remove sec r2 and remove pri r1,
=>state:{}
add new pri r3
=> {{r3,pri,5,0}}
set:simple_kv_open_fail=1 and add sec r2
=> {{r3,pri,5,0}}
group_check
=> state:{{r2,sec,6,0},{r3,pri,6,0}}
check r3 data
=> not-exist
change pri and check r2 data
=> not-exist
_
The same problem, so I restarted the downed replica and checked the data
=> state {{r1,pri,2,10},{r2,sec,2,10}}
remove sec r2
=>state{{r1,pri,2,10}}
add secondary r3 (new replica)
=>state:{{r1,pri,4,10},{r3,sec,4,10}}
write data #11-#15
=>state:{{r1,pri,4,15,15},{r3,sec,4,15,15}}
remove sec r3 and pri r1
=>state:{}
active pri r2 and check data
state:{{r2,pri,7,10,10}}
add sec r1 and set:simple_kv_open_fail=1
=> state:{{r1,pot,7,15,0},{r2,pri,7,10,10}} =>{{r2,pri,7,10,10}}
group_check
state:{{r1,sec,8,10},{r2,pri,8,10}}
 chenge pri and check data
state:{{r2,sec,10,10},{r1,pri,10,10}}
=> #1-#10 correct
=>#11-#15 not-exist
```

old-case-209:
```
=> state:{{r1,pri,2,10,10},{r2,sec,2,10,10}}
remove sec r2 and remove pri r1,
=>state:{}
add new pri r3
=> {{r3,pri,5,0}}
set:simple_kv_open_fail=1 and add sec r2
=> {{r3,pri,5,0}}
group_check
=> state:{{r2,sec,6,0},{r3,pri,6,0}}
check r3 data
=> not-exist
change pri and check r2 data
=> not-exist
_
The same problem, so I restarted the downed replica and checked the data
=> state {{r1,pri,2,10},{r2,sec,2,10}}
remove sec r2
=>state{{r1,pri,2,10}}
add secondary r3 (new replica)
=>state:{{r1,pri,4,10},{r3,sec,4,10}}
write data #11-#15
=>state:{{r1,pri,4,15,15},{r3,sec,4,15,15}}
remove sec r3 and pri r1
=>state:{}
active pri r2 and check data
state:{{r2,pri,7,10,10}}
add sec r1 and set:simple_kv_close_fail=1
=> state:{{r1,pot,7,15,0},{r2,pri,7,10,10}} =>{{r2,pri,7,10,10}}
group_check
state:{{r1,sec,8,10},{r2,pri,8,10}}
 chenge pri and check data
state:{{r2,sec,10,10},{r1,pri,10,10}}
=> #1-#10 correct
=>#11-#15 not-exist
```
The previous case is create a new replica when state: {}. Here is when state: {} active old replica, the others remain unchanged.hope your suggestion.